src/components/header/NavBar.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from 'react';
```

Comment:
This file name should match the folder name (wich name should start with a capital letter, the same as .scss file)